genmod/score_variants/score_variant.py

```diff
@@ -30,38 +30,52 @@ def get_plugin_score(variant, plugin_name, config_parser, csq_format=None):
     # Score is allways a number
     score = score_function.get_score(value)
     logger.debug("Score is {0} for plugin {1}".format(score, plugin_name))
+
+    plugin_score_min = score_function.score_min
+    plugin_score_max = score_function.score_max
+    logger.debug("Minmax scores for plugin {0} is ({1},{2})".\
+                  format(plugin_name, plugin_score_min, plugin_score_max))
     
-    return score
+    return score, plugin_score_min, plugin_score_max
 
 def get_category_score(variant, category, config_parser, csq_format=None):
-    """Return the score for a given category
-    
+    """Return the score for a given category.
+
+       A category (such as 'allele_frequency') groups multiple plugin scores.
+       This method selects final score based on category_aggregation [Categories]
+       rule (see genmod_example.ini)
+
         Args:
             variant (dict): A variant dictionary
             category (str): A score category
             config_parser (ConfigParser): A config parser object with score functions
         Returns:
             category_score (float): The rank score for this variant
     """
-    
+
     category_score = 0
-    
+
     logger.debug("Checking scores for category {0}".format(category))
     #We save all the scores for a category
     category_aggregate = config_parser.categories[category]['category_aggregation']
     category_scores = []
-    
+    category_score_min = 0.0
+    category_score_max = 0.0
+
     for plugin_name in config_parser.categories[category]['plugins']:
-        category_scores.append(get_plugin_score(
-            variant = variant, 
-            plugin_name = plugin_name, 
+        cs, plugin_score_min, plugin_score_max = get_plugin_score(
```

Comment:
What does cs stand for? `category_score`? It is better to be explicit, it helps readability and understanding.